nemo_skills/utils.py

```diff
@@ -128,7 +128,10 @@ def remove_handlers():
 
 
 def get_logger_name(file):
-    return "nemo_skills" + file.split("nemo_skills")[1].replace("/", ".").replace(".py", "")
+    if "nemo_skills" in file:
+        return "nemo_skills" + file.split("nemo_skills")[1].replace("/", ".").replace(".py", "")
+    else:
+        return Path(file).stem
```

Comment:
Should we consider adding a `nemo_skills.third_party.` prefix for cases where the file is outside the `nemo_skills` package?

Also, you might want to check for `/nemo_skills/` instead of `nemo_skills` for a hacky way to ensure it is a folder name and not just any substring.

Reply:
Also @dgtm777, you'll want to complete the commit sign-off.

Reply:
good idea about `/nemo_skills/`. For third_party, not sure, it looks as if that's just a module name and could be confusing. Either we should make it more explicit or just using a file name seems ok for now and we can refine it later